docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -119,6 +119,18 @@ This results in a reduction of the `@mui/material` package size by 2.5MB or 25%
 Instead, using ESM-based CDNs such as [esm.sh](https://esm.sh/) is recommended.
 For alternative installation methods, refer to the [CDN documentation](/material-ui/getting-started/installation/#cdn).
 
+### Autocomplete
```

Comment:
@DiegoAndai let me know if you think we should expand on this deprecation entry.